app/helpers/physical_storage_helper/textual_summary.rb

```diff
@@ -75,7 +75,11 @@ def textual_machine_type
   end
 
   def textual_model
-    {:label => _("Model"), :value => @record.asset_detail&.model}
+    if @record.ext_management_system.kind_of?(ManageIQ::Providers::Autosde::StorageManager)
```

Comment:
We really shouldn't have provider subclasses in core repositories if we can avoid it.

@agrare Thoughts here?

Reply:
Yeah :-1: on this, you can set the `asset_detail.model` in your Refresher to the name of the family which would then cause this to display correctly.

Reply:
changed as you requested ,
I opened a PR in the provider repo:
https://github.com/ManageIQ/manageiq-providers-autosde/pull/205